app/formdata/form_data_integration_test.go

```diff
@@ -471,7 +471,7 @@ func TestFormData_ConstructMapForDB(t *testing.T) {
 			map[string]interface{}{},
 		},
 		{
-			"skips unexported fields",
+			"skips fields without json tag",
```

Comment:
So maybe you should export it (uppercase) so that you do not test 2 things at the same time.

Reply:
We don't skip unexported fields anymore, so this test checks only one thing.

Reply:
This test differs in 2 points from other tests:
* it has no tags
* "name" is not exported

So your test, as it is, does show the effect of either one or the other, without guaranteeing that it is because of "no tags" that we get this result.

Reply:
improved the test

Reply:
Ok, that's better, but why still doing two tests in the same test. Reading the code, I don't know if unexported attributes are converted or not. 😕
If you want to put unexported attribute is a test, that's a good idea (that's why the initial test was about that), just do a one test with an unexported attribute and another one with no tags.

Sorry to bother you with such details 😞

Reply:
fixed :)